pkg/labels/selector.go

```diff
@@ -809,3 +809,77 @@ func SelectorFromSet(ls Set) Selector {
 func ParseToRequirements(selector string) ([]Requirement, error) {
 	return parse(selector)
 }
+
+// Conflicts takes 2 maps
```

Comment:
yeah because that is not a conflict, only same keys cause conflict so iterating on one map is fine.